addons/web_editor/static/src/js/wysiwyg/wysiwyg_iframe_content.js

```diff
@@ -0,0 +1,65 @@
+/** @odoo-module **/
+
+export function getWysiwygIframeContent(params) {
```

Comment:
Is there a particular reason why this function is alone in its own file even though it is imported only once ?

Reply:
I'm kind of afraid to resolve this comment as I'm not sure it's been done yet or if this is some kind of reminder...